docs/conf/portland/2020/news/covid-19-update-01.rst

```diff
@@ -0,0 +1,19 @@
+:template: {{year}}/generic.html
+
+.. post:: March 3, 2020
+   :tags: portland-2020, covid-19
+
+COVID-19 Update
+===============
+
+Hey Folks,
+
+As you've probably heard by now, the coronavirus (also known as COVID-19) is a new virus that causes respiratory illness in people and can easily spread from person-to-person. Since Write the Docs Portland 2020 is only a few months away, we wanted to give you an update on our status regarding COVID-19.
```

Comment:
```suggestion
As you've probably heard by now, the coronavirus (also known as COVID-19) is a new virus that causes respiratory illness in people and can easily spread from person-to-person. Since Write the Docs Portland 2020 is only 2 months away, we wanted to give you an update.
```